tests/checks/fixtures/disk/centos-df-Tk

```diff
@@ -0,0 +1,6 @@
+Filesystem Type 1K-blocks Used Available Use% Mounted on
+/dev/sda3 ext4 93421512 70963180 17706052 81% /
+tmpfs tmpfs 32969616 0 32969616 0% /dev/shm
+/dev/sda1 ext4 512752 90460 395412 19% /boot
+10.1.5.223:/vil/cor
+nfs 1020054752 56080768 963973984 6% /cor
```

Comment:
Nitpick: the output seems to be aligned by column, it might be good to make sure the fixture has all the extra white spaces this implies (see https://github.com/DataDog/dd-agent/blob/a6812459c9894378a4e5a1a3252cd49c3e9f6679/tests/test_system.py#L39-L51)